### PR TITLE
widget with safety disclaimer

### PR DESCRIPTION
**Pull request by hikmet-demir**

### CODE REVIEW

File: conversational-ai-sdks/widget.js [hunk 1]
```diff
@@ -9,5 +9,33 @@
...
+        <label>
+          <input required="required" type="checkbox" />
+          I consent to sharing my voice and data derived from my voice with 
+          third-party service providers to train and improve our customer 
+          support models.
+        </label>
+        <label>
+          <input required="required" type="checkbox" />
+          I understand that if I do not consent to the collection as 
+          described above, ElevenLabs [x] services cannot be provided to
```
Comment (am-holt) [inline conversational-ai-sdks/widget.js:35]: Do we need to specify which services?

Comment (am-holt) [reply conversational-ai-sdks/widget.js:35]: Is the [x] a placeholder?

Comment (hikmet-demir) [reply conversational-ai-sdks/widget.js:35]: no let me remove that x, the only service they can get over widget is convai

File: conversational-ai-sdks/widget.js [hunk 2]
```diff
@@ -9,5 +9,33 @@
...
+        <label>
+          <input required="required" type="checkbox" />
+          I consent to sharing my voice and data derived from my voice with 
+          third-party service providers to train and improve our customer 
+          support models.
+        </label>
+        <label>
+          <input required="required" type="checkbox" />
+          I understand that if I do not consent to the collection as 
+          described above, ElevenLabs services cannot be provided to
```
Comment (am-holt) [inline conversational-ai-sdks/widget.js:35]: Should we explicitly say, elevenlabs conversational widget services

Comment (am-holt) [reply conversational-ai-sdks/widget.js:35]: If a user doesn't accept these terms they can still get access to other elevenlabs services via our website right? Maybe I'm being pedantic. Did legal provide above terminology

Comment (hikmet-demir) [reply conversational-ai-sdks/widget.js:35]: I would prefer to leave it like this to be more broader, this is the text we got from legal directly

**Review by am-holt — APPROVED** [review]

**Comment by hikmet-demir** [comment]
will merge tomorrow after release

**Pull request merged**